features/aggregate_build_logs.feature

```diff
@@ -0,0 +1,10 @@
+@wip
+Feature: Aggregating Build Logs
+    Scenario: Browsing logs of container images builds during Amun inspection jobs
+        Given amun service is accessible using HTTPS
```

Comment:
@fridex @pacospace When I run `THOTH_USER_API_HOST=stage.thoth-station.ninja THOTH_MANAGEMENT_API_HOST=management.stage.thoth-station.ninja THOTH_AMUN_API_HOST=amun.stage.thoth-station.ninja pipenv run behave --tags=wip` to check only this feature, I get the error below:
```
@wip
Feature: Aggregating Build Logs # features/aggregate_build_logs.feature:2

  Scenario: Browsing logs of container images builds during Amun inspection jobs  # features/aggregate_build_logs.feature:3
    Given amun service is accessible using HTTPS                                  # features/steps/amun_api.py:39 0.001s
      Traceback (most recent call last):
        File "/home/bdauletb/.local/share/virtualenvs/integration-tests-PN0lkqBU/lib/python3.6/site-packages/behave/model.py", line 1329, in run
          match.run(runner.context)
        File "/home/bdauletb/.local/share/virtualenvs/integration-tests-PN0lkqBU/lib/python3.6/site-packages/behave/matchers.py", line 98, in run
          self.func(context, *args, **kwargs)
      TypeError: amun_accessible() got an unexpected keyword argument 'scheme'
```
Could you pls take a look?

Reply:
Looks like this line `@given('amun service is accessible using "{scheme}"')` solved it :smile: , thanks